cpp/src/Ice/OSLogLoggerI.cpp

```diff
@@ -10,48 +10,48 @@
 using namespace std;
 using namespace Ice;
 
-Ice::OSLogLoggerI::OSLogLoggerI(const std::string& prefix) : _prefix(prefix)
+Ice::OSLogLoggerI::OSLogLoggerI(string prefix) : _prefix(std::move(prefix))
 {
-    const string subsystem = prefix.empty() ? "com.zeroc.ice" : "com.zeroc.ice." + prefix;
+    const string subsystem = _prefix.empty() ? "com.zeroc.ice" : "com.zeroc.ice." + _prefix;
```

Comment:
Not directly related, but we should probably make subsystem a member, we use it in trace too.

Reply:
Fixed